armadillo/src/main/java/org/molgenis/armadillo/config/DataShieldConfigProps.java

```diff
@@ -0,0 +1,19 @@
+package org.molgenis.armadillo.config;
+
+import java.util.List;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@ConfigurationProperties(prefix = "datashield")
+@Component
+public class DataShieldConfigProps {
+  private final List<ProfileConfigProps> profiles;
+
+  public DataShieldConfigProps(List<ProfileConfigProps> profiles) {
```

Comment:
Some validation that a default profile is getting defined here and that the environments mentioned actually do exist. (But where?)